### DISCUSSION TITLE
Can we use alias for icons?

### DISCUSSION THREAD

**Discussion by jerrythomas** [unocss/unocss#1357] [Q&A]
Is it possible to provide aliases for icons from different collections when using preset-icons?

for example, instead of 
```
<i class="i-carbon-star" />
```

Is it possible to do 
```
<i class="rating-empty" />
```

**Comment by SnowingFox** [comment]
yes, you can use `shortcuts`

```js
shortcuts: {
    'rating-empty': 'i-carbon-star'
}
```

**Reply by jerrythomas** [reply]
I tried shortcuts. It didn't display the icon.

**Comment by zyyv** [comment]
Or you can custom your collection name to use. [#customization](https://github.com/unocss/unocss/blob/main/packages/preset-icons/README.md#customization)

**Reply by jerrythomas** [reply]
It's not very clear how to change the name using customization. The example shows either embedding the svg or pulling in the entire repo. I would like to mix a few icons from different packs and use them in the app with some generic names.

```
collections: {
        custom: {
          circle: '<svg viewBox="0 0 120 120"><circle cx="60" cy="60" r="50"></circle></svg>',
          /* ... */
        },
        carbon: () => import('@iconify-json/carbon/icons.json').then(i => i.default as any),
        /* ... */
      }
```

**Reply by userquin** [reply]
https://stackblitz.com/edit/sveltejs-kit-template-default-wnmf33?file=vite.config.js,src%2Froutes%2F__layout.svelte

check `src/routes/__layout.svelte` and the `vite.config.js`

first UnoCSS logo using `i-logos:unocss` and last one using the shortcut `unocss-logo`.

**Reply by jerrythomas** [reply]
Thank you, Joaquín. I had put the `shortcuts` inside the `theme {}`. Your example helped me realize my mistake.
